app/models/list.rb

```diff
@@ -0,0 +1,10 @@
+class List < ActiveRecord::Base
+  has_many :book_lists, dependent: :destroy
+  has_many :books, through: :book_lists
+
+  validates :title, :description, presence: true
+
+  def five_books
+    books.limit 5
```

Comment:
From our code review: change to `books[0..4]` to avoid extra query when calling `.limit`